ocaml/quicktest/quicktest.ml

```diff
@@ -902,7 +904,7 @@ let _ =
           maybe_run_test "powercycle" (fun () -> with_vm s vm_powercycle_test);
           maybe_run_test "lifecycle" (fun () -> with_vm s Quicktest_lifecycle.test);
           maybe_run_test "copy" (fun () -> Quicktest_vdi_copy.start s sr);
-          maybe_run_test "import_raw_vdi" (fun () -> Quicktest_import_raw_vdi.start s);
```

Comment:
Why this change?

Reply:
Ah, this happened because when I tested on my dtbox I commented out some of the tests so I could only run the relevant ones, I must've removed the semicolon while doing that, can undo it. Although I'm not sure why we include one for the final line within a try-with anyway, is it any different with or without?

Reply:
If it already has a semicolon you get less changes when people add stuff to the end: you only need to add a new line, not modify the old to add the ; and add new line. As discussed in another PR the preferred style is to put the ; at the beginning of the line, but lets follow what the existing code already does here.